x-pack/plugins/secops/public/components/timeline/body/renderers/index.ts

```diff
@@ -39,17 +40,17 @@ export * from './unknown_column_renderer';
 export * from './zeek_row_renderer';
 
 export const rowRenderers: RowRenderer[] = [
-  zeekRowRenderer,
-  suricataRowRenderer,
   auditdExecutedRowRenderer,
   auditdLoggedinRowRenderer,
   auditdWasAuthorizedRowRenderer,
   auditAcquiredCredsRowRenderer,
   auditdEndedSessionRowRenderer,
   auditDisposedCredsRowRenderer,
   auditdStartedSessionRowRenderer,
-  plainRowRenderer,
-];
+  netflowRowRenderer,
+  suricataRowRenderer,
+  zeekRowRenderer,
+].concat(plainRowRenderer); // falls-back to the plain row renderer
```

Comment:
I don't know what the significance of changing this to being a `concat` is and the changes with this ordering other than to maybe make this look different for others to notice that plainRowRenderer should always be here and that it is the fall back?

Unless you want to alphabetize this array except for plainRowRenderer.

If that is the case, it should be:

```
auditAcquiredCredsRowRenderer,
auditDisposedCredsRowRenderer,
auditdEndedSessionRowRenderer,
auditdExecutedRowRenderer,
auditdLoggedinRowRenderer,
auditdStartedSessionRowRenderer,
auditdWasAuthorizedRowRenderer,
netflowRowRenderer,
suricataRowRenderer
zeekRowRenderer,
```

You can also use a spread too which would get you ES2015/ES6 points instead of the concat fwiw

```ts
zeekRowRenderer,
... plainRowRenderer // falls-back to the plain row renderer but please keep the rest of these in lexicographical alphabetical order
]
```

I will change it in my PR since I redid all the audits